lib/heartcheck/generators/templates/config.rb

```diff
@@ -64,4 +64,10 @@
   monitor.add :cache do |c|
     c.add_service(name: 'memcached', connection: Rails.cache.instance_variable_get('@data'))
   end
+
+  # WatchFile
+  # check if a preloaded file as changed from the disk
+  monitor.add :watch_file do |c|
+    c.add_service(file: 'version.json')
```

Comment:
Could this file be configurable? It could be extended to other purposes and files.